resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -62,49 +137,300 @@
+                "ec2:DescribeIpv6Pools",
+                "ec2:DescribePublicIpv4Pools",
+                "ec2:GetAssociatedIpv6PoolCidrs"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "InternetGateway",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeEgressOnlyInternetGateways",
+                "ec2:DescribeInternetGateways"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "Describetag",
+            "Effect": "Allow",
+            "Action": [
+               "ec2:DescribeTags" 
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "PrefixLists",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribePrefixLists",
+                "ec2:DescribeManagedPrefixLists"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "PrefixListsentries",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:GetManagedPrefixListEntries"
+            ],
+            "Resource": "arn:aws:ec2:*:*:prefix-list/*"
+        },
+        {
+            "Sid": "Image",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeImages"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "ImagaeAttibute",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeImageAttribute"
+            ],
+            "Resource": "arn:aws:ec2:*:*:image/*"
+        },
+        {
+            "Sid": "IdFormat",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribePrincipalIdFormat",
+                "ec2:DescribeIdentityIdFormat",
+                "ec2:DescribeIdFormat",
+                "ec2:DescribeAggregateIdFormat"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "AddressAttribute",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeAddressesAttribute"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "Addresses",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeAddresses"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "AccountAttributes",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeAccountAttributes"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "ByoipCidrs",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeByoipCidrs"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "CapacityReservation",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeCapacityReservations"
+            ],
+            "Resource": "*"
```

Comment:
Any way these can be restricted more?

Reply:
doesn't seem like it when cross-checked with https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html#amazonec2-policy-keys

Reply:
And it doesn't seem like SRE would use this permission so will remove this